schemas/file.yml

```diff
@@ -184,3 +184,21 @@
         https://www.iana.org/assignments/media-types/media-types.xhtml[IANA official types],
         where possible. When more than one type is applicable, the most specific
         type should be used.
+
+    - name: fork_name
+      level: extended
+      type: keyword
+      description: >
+        A fork is additional data associated with a filesystem object.
```

Comment:
Thanks for the detailed description. 👍 

This type of "real-world usage" detail here would be great to capture in a `file.*` usage doc, similar to what was done with [user](https://www.elastic.co/guide/en/ecs/current/ecs-user-usage.html).

Not necessary for this addition, though, but a note for when it's revisited. 😃

Reply:
Good point. Yeah this info spans multiple fields so it makes sense to capture that at a higher level.

There's still work to do on the PR and some cleanup in my end, so I'm gonna do another round or two of edits